functions/pdf.js

```diff
@@ -31,14 +31,39 @@ export async function render(chat, ego, isSample = false) {
 
   const backgroundGreenHex = "#21a68d";
 
+  const lastDate = chat.chatObject.slice(-1)[0].date;
```

Comment:
Duplice Code:

dateDiffs in TextStats?

Reply:
Ja ich will anzeigen wie viele Tage die siech kennen quasi auf Whats App

das ist nirgends gespeichert aber da einfach ausrechenbar